tests/python/frontend/oneflow/test_forward.py

```diff
@@ -359,7 +359,7 @@ def forward(self, x):
     model = Conv2dModel()
     model.eval()
 
-    for device in ["llvm", "cuda"]:
```

Comment:
You can (and should) test cuda on TVM, just need to compare against the CPU reference from oneflow.

Reply:
OK, I just updated，and [this pr](https://github.com/apache/tvm/pull/9922) changed oneflow to cpu installation.

Reply:
I didn't see the update? Still only llvm is being tested.